src/CarrierLogo/CarrierLogo.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View, Image, PixelRatio } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import StyleSheet from '../PlatformStyleSheet';
+import type { Props } from './CarrierLogoTypes';
+import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS, BASE_URL } from './consts';
+import { logoSizes } from './styles';
+
+export default function CarrierLogo({
+  size = SIZE_OPTIONS.LARGE,
+  carriers = [],
+}: Props) {
+  const carriersLength = carriers.length;
+  const urlSize =
+    (carriersLength > 1 || size === SIZE_OPTIONS.SMALL ? 16 : 32) *
+    (PixelRatio.get() >= 2 ? 2 : 1);
```

Comment:
Could you explain why the PixelRatio will always be maximum 2?

Reply:
This way I'm handling just retina/non-retina displays.
But I see that there is even bigger version of assets available, so I'll edit and extend it even for `@3x` cases.